models/enterprise_sonic/vrrp/overridden_example_01.txt

```diff
@@ -0,0 +1,96 @@
+# Using overridden
+#
+# Before State:
+# -------------
+#
+#sonic# show running-configuration interface
+#!
+#interface Ethernet0
+# mtu 9100
+# speed 400000
+# fec RS
+# no shutdown
+# ip address 81.1.1.1/24
+# ipv6 address 81::1/24
+# !
+# vrrp 1 address-family ipv4
+# preempt
+# vip 81.1.1.3
+# vip 81.1.1.4
+# !
+# vrrp 10 address-family ipv6
+# priority 10
+# advertisement-interval 4
+# vip 81::3
+# vip 81::4
+#!
+#interface Ethernet8
+# mtu 9100
+# speed 400000
+# fec RS
+# no shutdown
+# ip address 61.1.1.1/24
+# !
+# vrrp 5 address-family ipv4
+# priority 20
+# vip 61.1.1.3
+# !
+# vrrp 15 address-family ipv4
+# priority 20
+# preempt
+# vip 61.1.1.4
+#!
+
+  - name: Overwrite the VRRP and VRRP6 relay configurations
+    sonic_vrrp:
+      config:
+        - name: 'Eth1/1'
+          group:
+            - vrrp_id: 1
+	      type: ipv4
+	      vip: 
+		- address: 81.1.1.5
+	      preempt: False
+	    - vrrp_id: 10
+	      type: ipv6
+        - name: 'Eth1/3'
+          group:
+            - vrrp_id: 5
+	      type: ipv4
+	    - vrrp_id: 15
+	      type: ipv4
+	      vip: 
+		- address: 61.1.1.5
+      state: overriden
+
+# After State:
+# ------------
+#
+#sonic# show running-configuration interface
+#!
+#interface Ethernet0
+# mtu 9100
+# speed 400000
+# fec RS
+# no shutdown
+# ip address 81.1.1.1/24
+# ipv6 address 81::1/24
+# !
+# vrrp 1 address-family ipv4
+# no preempt
```

Comment:
I experimented with this and I see that "no preempt" does show up in the "show running-configuration" output for this case. (This is strange, because default configuration settings are usually not displayed in "show running-configuration".) Nevertheless, since this does match the current implementation, we can consider this issue "Resolved".